cmd/docker-app/merge.go

```diff
@@ -50,7 +51,15 @@ func mergeCmd(dockerCli command.Cli) *cobra.Command {
 				return err
 			}
 			defer extractedApp.Cleanup()
-			inPlace := mergeOutputFile == ""
+			inPlace := false
```

Comment:
duplicate with `split` below, we could extract it to a func.

Reply:
Agree that the code is near identical and this could be factored. Not immediately clear how to name such a function or if we need to do a bigger refactor for this to make sense.